war/src/main/webapp/css/style.css

```diff
@@ -522,6 +522,10 @@ table.bigtable.pane > tbody > tr > td:last-child {
   white-space: nowrap;
 }
 
+.bigtable th.minimum-width {
+  width: 1px;
```

Comment:
Tested with Firefox, Chrome, Opera, IE 11, the table is expanded to the minimum width required by the content.